### PR TITLE
mgr/dashboard: Evict a CephFS client

### PR DESCRIPTION
**Pull request by ricardoasmarques**
It's now possible to evict (disconnect) a CephFS client, from the UI:

c89bb0fa-5952-45d8-a66d-ad8f01f8adc0

Fixes: https://tracker.ceph.com/issues/24892

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.spec.ts [hunk 1]
```diff
@@ -4,7 +4,15 @@ import { RouterTestingModule } from '@angular/router/testing';
 
 import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
 
-import { configureTestBed, i18nProviders } from '../../../../testing/unit-test-helper';
+import { By } from '@angular/platform-browser';
+import { ToastModule } from 'ng2-toastr';
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.spec.ts]: We are now using a different package.
```suggestion
import { ToastrModule } from 'ngx-toastr';
```

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.ts [hunk 2]
```diff
@@ -14,10 +23,30 @@ export class CephfsClientsComponent implements OnInit {
...
+    private modalService: BsModalService,
+    private notificationService: NotificationService,
+    private authStorageService: AuthStorageService,
+    private i18n: I18n,
+    private actionLabels: ActionLabelsI18n
+  ) {
+    this.permission = this.authStorageService.getPermissions().cephfs;
+    const evictAction: CdTableAction = {
+      permission: 'update',
+      icon: 'fa-sign-out',
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.ts]: Please use the new Icons class
```suggestion
      icon: Icons.xxx,
```

**Comment by ricardoasmarques** [comment]
@tspmelo Comments addressed, thanks.

**Comment by ricardoasmarques** [comment]
jenkins retest this please

**Pull request merged**